app/konnect/api-products/productize-service.md

```diff
@@ -41,16 +62,69 @@ After creating the new version, you will see **Link with a Gateway Service** as
     Choose the [control plane](/konnect/gateway-manager/control-plane-groups/) and [Gateway Service](/konnect/gateway-manager/configuration/#gateway-services) to
     deploy this API product version to. This lets you deploy your service across data plane nodes associated with the control plane.
 1. Click **Save**.
-
+{% endnavtab %}
+{% navtab API%}
+
+1. To create a new API product version, execute a POST request to the  [`/api-product-versions/`](/konnect/api/api-products/latest/#/API%20Product%20Versions/create-api-product-version) endpoint, replace `{ApiProductId}` with your API product's actual ID:
```

Comment:
should this be `/product-versions/` instead of `/api-product-versions/`?

Reply:
Yup! I fixed it, nice catch.